aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/ListHandler.java

```diff
@@ -133,7 +141,11 @@ private DescribePatchBaselinesResponse describePatchBaselinesResponse(final Stri
                     = readHandler.handleRequest(proxy, requestFromId, null, logger);
 
             ResourceModel resourceModel = response.getResourceModel();
-            models.add(resourceModel);
+
+            if(resourceModel != null){
```

Comment:
what could cause the case of resourceMode to be null?

Reply:
??

Reply:
If the resource model is null and the process is trying to add the resource model object we are adding to LIST object now.. 
When try to get those details, in the LIST the process will get NUllPointerException and process will get teriminated.
To avoid that error cause i have handle the null check before adding the null values to the list.

If you have any conerns please let me know.

Reply:
okay, as long as it actually has some usage

Reply:
okay fine